.github/support-packages/create-metadata-dir.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+for version_dir in /tmp/support-packages/metadata/collector-versions/*; do
+    version="$(basename "$version_dir")"
+    git checkout "${version}"
+
+    driver_version="$(cat "${GITHUB_WORKSPACE}/kernel-modules/MODULE_VERSION")"
+    echo "${driver_version}" > "${version_dir}/MODULE_VERSION"
```

Comment:
I am curious about these two lines. This could probably be written as a `cp` call. Or maybe the intent is to do some checking ?

Reply:
No, you are right, I think a `cp` command would suffice, I was just doing mindless copy pasting, I'll fix it.